reference/pbrappearance.md

```diff
@@ -0,0 +1,70 @@
+## PBRAppearance
+
+```
+PBRAppearance {
+  SFColor  baseColor            1 1 1             # any color
+  SFNode   baseColorMap         NULL              # {ImageTexture, PROTO}
+  SFFloat  transparency         0                 # [0, 1]
+  SFFloat  roughness            0                 # [0, 1]
+  SFNode   roughnessMap         NULL              # {ImageTexture, PROTO}
+  SFFloat  metalness            1                 # [0, 1]
+  SFNode   metalnessMap         NULL              # {ImageTexture, PROTO}
+  SFNode   environmentMap       NULL              # {Cubemap, PROTO}
+  SFFloat  IBLStrength          1                 # [0, inf)
+  SFNode   normalMap            NULL              # {ImageTexture, PROTO}
+  SFFloat  normalMapFactor      1                 # [0, inf)
+  SFNode   occlusionMap         NULL              # {ImageTexture, PROTO}
+  SFFloat  occlusionMapStrength 1                 # [0, inf)
+  SFNode   textureTransform     NULL              # {TextureTransform, PROTO}
+  SFString name                 "PBRAppearance"   # any string
+}
+```
+
+### Description
+
+%figure "An Oxidised Copper Sphere made with the PBRAppearance node"
+
+![pbrappearance.png](images/pbrappearance.png)
+
+%end
+
+The [PBRAppearance](#pbrappearance) node specifies a a physically-based visual appearance of a node.
+The acronym "PBR" refers to "Physically-Based Rendering", a term used to designate a class of shading models based on the physical properties of an object, as opposed to approximations.
```

Comment:
> as opposed to approximations.

I am not sure 'approximations' is the correct term to describe our phong shader because even if PBR is way more realistic it is still an approximation.